### PR TITLE
Feat[TE-15569]: Addon to access downloaded PDF files on the cloud and extract the content

### PR DESCRIPTION
**Pull request by SunilGembali**

### CODE REVIEW

File: pdfdataextractor/src/main/java/com/testsigma/addons/web/util/PdfAndDocUtilities.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        String remoteFilePath = null;
+        if(fileName != null){
+            remoteFilePath = getFilePathByFileNameInDownloads(fileName);
+        } else {
+            remoteFilePath = getDownloadedFileLocalPath();
+        }
+        logger.info("Downloaded file path="+remoteFilePath);
+        File downloadedFile = createLocalFileFromDownloadsCopy(remoteFilePath,fileFormat);
+        //switch to parent window tab
+        driver.switchTo().window(currentWindowHandle);
```
Comment (rajesh-testsigma) [inline pdfdataextractor/src/main/java/com/testsigma/addons/web/util/PdfAndDocUtilities.java]: Move this to finally, this is mandatory switch

Comment (SunilGembali) [reply pdfdataextractor/src/main/java/com/testsigma/addons/web/util/PdfAndDocUtilities.java]: Moved to finally block.

**Review by ManoharKrishnaBoppe — CHANGES_REQUESTED** [review]
Please do update the kibbutz bootstrap.

**Comment by SunilGembali** [comment]
> Please do update the kibbutz bootstrap.

I made the changes and raised the PR for testsigma-kibbutz

**Review by rajesh-testsigma — APPROVED** [review]

**Pull request merged**